config/mbrave.yml

```diff
@@ -0,0 +1,206 @@
+---
+default: &DEFAULT
+  bioscan_p1:
+    version: 1
```

Comment:
there is already a version 2, so maybe we should start with that version (see Scotts slack message about what changed)

Reply:
We are starting with version 2 now. The tag name changes for version 2 have been added. Reverse grouping is has been merged.